src/platform_impl/linux/wayland/window/state.rs

```diff
@@ -840,6 +850,20 @@ impl WindowState {
         }
     }
 
+    /// Make window background blurred
+    #[inline]
+    pub fn set_blur(&mut self, blurred: bool) {
+        if blurred {
```

Comment:
Is it really fine to do it like that? I think we need to update only when blur changes the state and have initial state as false? Or we could simply destroy the `blur` object?